x-pack/plugins/infra/public/utils/fetch.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import axios from 'axios';
+const FETCH_TIMEOUT = 30000;
+
+export const fetch = axios.create({
+  headers: {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+    'kbn-xsrf': 'professionally-crafted-string-of-text',
```

Comment:
Is this supposed to be set to `professionally-crafted-string-of-text`? (I know nothing of the `kbn-xsrf` header 😅)

Reply:
It just needs to be a static string... I copied part of that from Canvas and I'm 100% sure that Rashid put that string in there which I thought was funny, so of course I had to leave it.